apps/src/lab2/lab2Redux.ts

```diff
@@ -90,7 +89,7 @@ export const setUpWithLevel = createAsyncThunk(
   ) => {
     try {
       // Update properties for reporting as early as possible in case of errors.
-      Lab2MetricsReporter.updateProperties({
+      Lab2Registry.getInstance().getMetricsReporter().updateProperties({
```

Comment:
Although longer, I appreciate the clarity of this change as someone who is not as familiar with lab2.